pages/doc/logging_drill_into_logs.md

```diff
@@ -0,0 +1,169 @@
+
+{% include note.html content="You must have tagged the traces and the logs from the same applications and services with equivalent application and service tag values. If your traces and logs tags don't match, users with **Accounts** permissions can map the traces tags to logs tags. For details, see [Customize Logs Settings](logging_logs_settings.html)."%}
+
+* **From the Map View**
+  1. Select the time window of interest.
+  1. Click the service on the application map.
+  1. Select **View Logs (Beta)**.
+  ![A screenshot of a the UI once you click on a service with the view logs link highlighted.](images/logging_app_map_to_logs.png)
+* **From the Table View**
+  1. Select the time window of interest.
+  1. Click the ellipsis for the service.
+  1. Select **View Logs (Beta)**.
+  ![A screenshot of a the UI once you click vertical ellipsis on the table view](images/logging_table_view_to_logs.png)
+* **From the Grid View**
+  1. Select the time window of interest.
+  1. In a service tile, click **Actions**.
+  1. Select **View Logs (Beta)**.
+  ![A screenshot of a the UI once you click vertical ellipsis on the grid view](images/logging_grid_view_to_logs.png)
+
+The Logs Browser opens in a new tab with the following configurations:
+  
+* The search time window corresponds to the time window on the Application Status page.
+* The search query contains the corresponding include `service` and `application` tag filters.
+![The search query and the selected time window in the Logs Browser.](images/logging_app_serv_search.png)
+
+### Traces Browser
+
+If you notice a critical path through a trace in the [Traces Browser](tracing_traces_browser.html), you can drill down into the related logs.
+
+{% include note.html content="You must have tagged the traces and the logs from the same sources, applications, and services with equivalent source, application, and service tag values. If your traces and logs tags don't match, users with **Accounts** permissions can map the traces tags to logs tags. For details, see [Customize Logs Settings](logging_logs_settings.html)." %}
+
+To see the logs for a trace:
+1. Click the trace that you want to examine.
+1. In the Trace Details section, click the service on which you want to focus.
+1. Expand the **IDs** section.
+1. Click **Search Logs (Beta)**.
+![screenshot of the traces browser with the search logs highlighted](images/logging_traces_browser.png)
+
+The Logs Browser opens in a new tab with the following configurations:
+  
+* By default, the search time window starts 5 seconds before the trace and ends 5 seconds after the trace.
+  {% include note.html content="To change the default search time window for trace logs, contact [Technical Support](wavefront_support_feedback.html#support)."%}
+* The search query contains the corresponding include `traceId`, `source`, `application`, and `service` tag filters.
```

Comment:
The search query contains the corresponding `traceId`, `source`, `application`, and `service` tag filters included.